### PR TITLE
Bear Stuff

### PR DESCRIPTION
**Pull request by NanakoAC**
Changelog kept intentionally vague and mysterious

changes: 
  - bugfix: "Space bears can now control their movement in space."
  - rscadd: "Bears are now stronger in space or low pressure, weaker in pressurised environments"
  - soundadd: "Rawr!"
  - rscadd: "Added more bears!"
  - rscadd: "Added a new moderate and severe event."

Overhauls space bears a lot, fixing many bugs and issues with their AI, as well as their controlled space movement not working, and a bug where they would randomly get tired before they start attacking. All bears will now immediately aggro when harmed, so they're not so easy to just gun down before they can respond.

Adds lots of growly roary audio for bears. In general all bears are very loud and bad at stealth, making it easy to track them down via sound. Or to avoid, if you're not keen on being eaten today.

Bear space interaction tweaked a bit, they already changed icons depending on whether or not they were in space, i expanded it to provide some mechanical bonuses in their natural habitat.

Adds two new events, One moderate event spawns some bears that will nest in maintenance. They won't assault people that leave their territory quietly, so security can take their time hunting them down.

And adds a severe event,  bluespace bears escape from a nearby lab and wreak havoc. They're an experimental bioweapon and are highly mobile predators, demanding an active and intense security response. Bluespace bears are capable of long-range teleports around the station periodically until they find someone, and then, in combat, teleporting around the local area rapidly but randomly every few seconds, or when shot. They are also solitary hunters, refusing to cooperate. Whenever two of them meet, one will immediately teleport away, so security only ever has to deal with one at a time. 

Carefully tuned to provide a good challenge, they are best dealt with by a coordinated security response, using weapons with extreme singleshot damage, like laser cannons, slug shotguns and sniper rifles. Rapid fire weapons are ineffective as they'll usually teleport mid-burst and dodge most of the bullets.

Civilians can avoid harm by quietly moving away from any that they see, they will still watch and growl a bit before attacking, and won't chase if the target leaves before they get mad. Fleeing into the halls is also a good strategy, as their tactical teleporting is far less effective in large open areas

### CODE REVIEW

**Comment by skull132** [comment]
Bear event is also undergoing administrative review. It's close to the "Wtfbbq" area of SS13, and may not befit our server.

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 1]
```diff
@@ -73,100 +118,331 @@
...
+	name = "bluespace bear"
+	desc = "*bzzt*..Rawr!!"
+	maxHealth = 130
+	turns_per_move = 7
+	break_stuff_probability = 100//Constantly smashing everything nearby
+	speak_chance = 15
+	var/idletime
+	var/focus_time//How long we've focused on this target
+	var/teleport_delay = 60
+	var/tactical_delay = 3//Procs between shortrange teleports
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: `var/code // stuff` please. The formatting breaks are annoying to read.

File: code/modules/mob/living/simple_animal/hostile/bear.dm [hunk 2]
```diff
@@ -73,100 +118,331 @@
...
+	spawn(2)
+		qdel(s)
+
+/mob/living/simple_animal/hostile/bear/spatial/FindTarget()
+	. = ..()
+	if(target_mob)
+		focus_time = 0
 
 
+#undef BEARMODE_INDOORS
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/hostile/bear.dm]: :+1:

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 3]
```diff
@@ -115,7 +116,7 @@
 
 /mob/living/simple_animal/Life()
 	..()
-
+	life_tick++
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: Is this needed anywhere?

File: html/changelogs/Nanako-Bears.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Space bears can now control their movement in space."
+  - rscadd: "Bears are now stronger in space or low pressure, weaker in pressurised environments"
+  - soundadd: "Rawr!"
+  - rscadd: "Added more bears!"
```
Comment (skull132) [inline html/changelogs/Nanako-Bears.yml]: Clean up the log at some point as well. Please.

**Comment by NanakoAC** [comment]
RE: Administrative review, i created this to fill a major need - lack of variety in severe events

In terms of strangeness, i don't see how its much different to a giant blob, or a spider infestation, or space vines, or 200 carp spawning around the area. Something gets on the station and you have to deal with it

I've put a lot of effort into this, and i'm happy to make modifications as requested

**Comment by skull132** [comment]
Remove the moderate severity event.
Consider making the severe one asteroid focused, as discussed. 
And solve whatever conflicts there are.

Once that's done, I'll merge it.

**Comment by NanakoAC** [comment]
Moderate event removed. I'm really sad about this :<
Merge errors resolved

I'm not clear what is meant by 'asteroid-focused', but as mentioned the events are designed to be future proof and should work seamlessly in the new map

**Pull request merged**